docs/source/notebooks/gridded_data_subset.ipynb

```diff
@@ -15,28 +16,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# This entire section is cookie-cutter template to allow calling the servers and instantiating the connection\n",
-    "# to the WPS server. Do not modify this block.\n",
-    "\n",
     "from pathlib import Path\n",
-    "from urllib.request import urlretrieve\n",
     "from zipfile import ZipFile\n",
-    "import glob\n",
     "import json\n",
     "import os\n",
     "import datetime as dt\n",
     "import tempfile\n",
-    "\n",
     "from birdy import WPSClient\n",
-    "from matplotlib import pyplot as plt\n",
-    "from xclim import subset\n",
     "import fiona\n",
-    "import netCDF4 as nc\n",
-    "import numpy as np\n",
-    "import rioxarray\n",
```

Comment:
rioxarray is required for me, it fails without it. for clipping the data to the shapefile.